tests_e2e/test_suites/ext_cgroups.yml

```diff
@@ -10,4 +10,8 @@ images: "cgroups-endorsed"
 locations: "AzureCloud:southcentralus"
 skip_on_clouds:
   - "AzureChinaCloud"
-  - "AzureUSGovernment"
\ No newline at end of file
+  - "AzureUSGovernment"
+skip_on_images: # Skip arm64 images as DCR test extension is not published for arm64 images
+  - "rhel_95_arm64"
```

Comment:
We don't have any v1 arm64 images already being tested?

Reply:
Today we use these and none of them v1 distros

   endorsed-arm64:
      - "debian_11_arm64"
      - "flatcar_arm64"
      - "mariner_2_arm64"
      - "azure-linux_3_arm64"
      - "rhel_95_arm64"
      - "ubuntu_2204_arm64"
      - "ubuntu_2404_arm64"